kf_lib_data_ingest/target_api_plugins/kids_first_dataservice.py

```diff
@@ -429,6 +429,79 @@ def submit(cls, host, body):
         return submit(host, cls, body)
 
 
+class Sample:
+    class_name = "sample"
+    api_path = "samples"
+    target_id_concept = (
+        CONCEPT.SAMPLE.TARGET_SERVICE_ID
+        or CONCEPT.BIOSPECIMEN_GROUP.TARGET_SERVICE_ID
+    )
+    service_id_fields = {"kf_id", "participant_id"}
+
+    @classmethod
+    def get_key_components(cls, record, get_target_id_from_record):
+        return {
+            "study_id": get_target_id_from_record(Study, record),
+            "external_id": not_none(record[CONCEPT.BIOSPECIMEN_GROUP.ID]),
```

Comment:
@znatty22 just noticied I need to get this pointing to sample, not biospecimen_group

```suggestion
            "external_id": not_none(record[CONCEPT.SAMPLE.ID]),
```